xyz-hub-service/src/main/java/com/here/xyz/httpconnector/config/AwsS3Client.java

```diff
@@ -78,6 +80,15 @@ public URL generateUploadURL(String bucketName, String key) throws IOException {
     }
 
     public URL generatePresignedUrl(String bucketName, String key, HttpMethod method) throws IOException {
+
+        AmazonS3ClientBuilder builder = AmazonS3ClientBuilder.standard();
```

Comment:
Maybe it would be better to move the retrieval of credentials out of this method, because this method could be called multiple times / for each download URL after a job has finished, but the credentials retrieval only has to happen once before all these URLs are created.